src/bin/nydusd/main.rs

```diff
@@ -20,17 +20,19 @@ use std::convert::TryInto;
 use std::fs::File;
 use std::io::{Read, Result};
 use std::ops::DerefMut;
+use std::sync::Mutex;
 use std::sync::{
     atomic::{AtomicBool, Ordering},
     mpsc::channel,
-    Arc, Mutex,
+    Arc,
 };
-use std::thread;
+use std::thread::{self, spawn};
 use std::{io, process};
 
 use clap::{App, Arg};
 use event_manager::{EventManager, EventSubscriber, SubscriberOps};
```

Comment:
Are these still needed after switch to mio?

Reply:
"spawn" is still needed for starting monitor so far(line 496).

Reply:
The http thread is still using eventfd to exit itself. Are you planning to switch it (via the new micro_http crate) in a different PR?

Reply:
> The http thread is still using eventfd to exit itself. Are you planning to switch it (via the new micro_http crate) in a different PR?

Yes. In a later PR.